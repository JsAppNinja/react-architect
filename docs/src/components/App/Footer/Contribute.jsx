import React from 'react';
import { Button, Cell } from 'react-md';
import { GITHUB_URL } from 'constants/application';

const Contribute = () => (
  <Cell component="section" size={8}>
    <h4 className="md-title">Contributing</h4>
    <p className="md-body-2">
      This project is currently developed by a single person. Feel free
      to contribute!
    </p>
    <Button flat secondary href={GITHUB_URL} iconClassName="fa fa-github">
      Github
    </Button>
  </Cell>
);

export default Contribute;
