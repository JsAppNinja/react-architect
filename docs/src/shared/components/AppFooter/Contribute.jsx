import React from 'react';
import Button from 'react-md/lib/Buttons/Button';

import { GITHUB_URL } from 'constants/application';

const Contribute = () => (
  <section className="md-cell md-cell--8">
    <h4 className="md-title">Contributing</h4>
    <p className="md-body-2">
      This project is currently developed by a single person. Feel free
      to contribute!
    </p>
    <Button flat secondary href={GITHUB_URL} iconClassName="fa fa-github">
      Github
    </Button>
  </section>
);

export default Contribute;
