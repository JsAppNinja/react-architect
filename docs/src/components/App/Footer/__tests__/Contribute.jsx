/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import Contribute from '../Contribute';

describe('Contribute', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Contribute />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
