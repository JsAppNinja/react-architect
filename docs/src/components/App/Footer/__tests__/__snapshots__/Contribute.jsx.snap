// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contribute should render correctly 1`] = `
<section
  class="md-cell md-cell--8"
>
  <h4
    class="md-title"
  >
    Contributing
  </h4>
  <p
    class="md-body-2"
  >
    This project is currently developed by a single person. Feel free to contribute!
  </p>
  <a
    class="md-btn md-btn--flat md-btn--text md-pointer--hover md-text--theme-secondary md-ink--secondary md-inline-block"
    href="https://github.com/mlaursen/react-md"
  >
    <div
      class="md-ink-container"
    />
    <div
      class="md-icon-separator"
    >
      <i
        class="md-icon fa fa-github md-text--inherit"
      />
      <span
        class="md-icon-text"
      >
        Github
      </span>
    </div>
  </a>
</section>
`;
